src/components/LicenseRevokeModal/index.jsx

```diff
@@ -85,7 +86,7 @@ class LicenseRevokeModal extends React.Component {
         <div className="license-details mb-4">
           <React.Fragment>
             <p className="message" >Revoking a license will remove access to the subscription catalog
-              for {user.emailAddress}. To re-enable access, you can assign this user to another
+              for {user.user_email}. To re-enable access, you can assign this user to another
```

Comment:
Ideally, we would be camelCasing any API response so the object keys are not snake_case (see `camelCaseObject` in utils).